Netskope/netskope_modules/connector_pubsub_lite.py

```diff
@@ -0,0 +1,144 @@
+import asyncio
+import gzip
+import time
+from functools import cached_property
+
+from google.cloud.pubsublite.cloudpubsub import AsyncSubscriberClient
+from google.cloud.pubsublite.types import CloudRegion, CloudZone, FlowControlSettings, SubscriptionPath
+from pydantic import BaseModel
+
+from .base import AsyncGoogleTrigger
+from .metrics import FORWARD_EVENTS_DURATION, INCOMING_MESSAGES, OUTCOMING_EVENTS
+
+
+class PubSubLiteConfig(BaseModel):
+    project_id: str
+    cloud_region: str
+    zone_id: str | None = None
+    subscription_id: str
+
+    frequency: int = 20
+    intake_server: str = "https://intake.sekoia.io"
+    intake_key: str
+    chunk_size: int = 1000
```

Comment:
I suggest importing `DefaultConnectorConfiguration` from `sekoia_automation.connector` and use it:
```suggestion
class PubSubLiteConfig(DefaultConnectorConfiguration):
    project_id: str
    cloud_region: str
    zone_id: str | None = None
    subscription_id: str

    frequency: int = 20
    chunk_size: int = 1000
```